frame/staking/src/slashing.rs

```diff
@@ -411,12 +409,21 @@ fn slash_nominators<T: Config>(
 			let target_span = spans.compare_and_update_span_slash(params.slash_era, era_slash);
 
 			if target_span == Some(spans.span_index()) {
-				// End the span, but don't chill the nominator. its nomination
-				// on this validator will be ignored in the future.
+				// end the span, but don't chill the nominator.
 				spans.end_span(params.now);
 			}
 		}
 
+		// in any case, remove the nomination.
+		<Pallet<T> as Store>::Nominators::mutate(
+			&nominator.who,
+			|maybe_nomination: &mut Option<crate::Nominations<T>>| {
+				if let Some(nomination) = maybe_nomination.as_mut() {
+					nomination.targets.retain(|t| t != params.stash)
```

Comment:
maybe we should only chill if the slash percentage is more than a threshold, or generally a condition is met.

Reply:
generally, we want to, perhaps, not chill nominators, if the slash is zero (offline-ness)